Add initial design values to SimpleGADriver

madsmpedersen opened the pull request:
Add initial design variable values as first individual of first generation in SimpleGADriver

Kenneth-T-Moore commented on openmdao/drivers/genetic_algorithm_driver.py:
```diff
@@ -219,6 +219,16 @@ def run(self):
+            x0[i:i + size] = desvar_vals[name]
+            i += size
+
+
```
You can move this into the previous loop with the cached indices so that you don't have to sum the sizes again. So it'll look like x0[i:j] = desvar_vals[name].

Looks good otherwise. You might need to remove the extra carriage return here to pass the lint tests.

madsmpedersen commented:
God point - I have now implemented it.

In the ´test_two_branin_parallel_model´ test, the random population initialization was quite lucky, and therefore this test failed when new_gen[0] was overwritten with the initial independen_variable values.
The test passed if I increased the max_gen value from 10 to 20, but instead I changed the initial independent_variable value to a more lucky value (7.5 to 5)

Kenneth-T-Moore commented:
@madsmpedersen : Ok, I think that change to the test initial point is fine.
